frameworks/mbed_extra.rst

```diff
@@ -101,6 +101,74 @@ An example of :ref:`projectconf` with enabled ``rtos``
     build_flags = -D PIO_FRAMEWORK_MBED_RTOS_PRESENT
 
 
+Custom version of Mbed
+^^^^^^^^^^^^^^^^^^^^^^
+
+PlatformIO comes with only a few versions of Mbed supported out of the box. If you want
```

Comment:
I'd change this wording to something like "PlatformIO only has certain versions of mbed-os in the official package package registry, as queriable through [the API](https://api.registry.platformio.org/v3/packages/platformio/tool/framework-mbed)", just to give people a pointer as to where to find natively supported versions.